packages/api/src/sdk-bundle-api/sdk-to-bundle/network-stubbing.ts

```diff
@@ -53,3 +61,41 @@ export interface ConnectionTypesFilter {
   xhr: boolean;
   webSockets: boolean;
 }
+
+type TransformableRequestData = Pick<Request, "body" | "method" | "url">;
+
+interface CustomTransformationBase {
+  /**
+   * The string to search for in the request component. This can also be a regular expression.
```

Comment:
```suggestion
   * The regex to match against the request component.
```

(if you want to search a string I think you'd need to write it as a regex and so escape it)

Reply:
(maybe matchRegex or something would be a better name given this)